netconfdriver/service/operations/config_operations.py

```diff
@@ -30,14 +30,48 @@ def netconf_connect(host, port, user, password,rsa_key_path,kwargs):
     def netconf_edit(conn,package_properties,default_operation,request_id):
         try:
             external_request_id = str(uuid.uuid4())
-            ConfigOperations._generate_additional_logs(package_properties, 'sent', external_request_id,
-                                       'message', 'rpc', {'operation' : default_operation}, request_id)
+            ConfigOperations._generate_additional_logs(package_properties, 'sent', external_request_id, 'application/xml',
+                                       'request', 'netconf', {'default-operation' : default_operation}, request_id)
             edit_config_details = conn.edit_config(config=package_properties, target="running", default_operation=default_operation)
-            ConfigOperations._generate_additional_logs(edit_config_details, 'received', external_request_id,
-                                       'message', 'rpc', {'operation' : default_operation}, request_id)
+            response_err = ""
+            if edit_config_details.error == None:
+                    response_err =''
+            else:
+                response_err = edit_config_details.error
+             
```

Comment:
The above if else condition code can be simplified with following lines.
response_err = ''
if edit_config_details.error != None:
   response_err = edit_config_details.error